backend/pkg/api/handlers/common.go

```diff
@@ -236,6 +245,21 @@ func (v *validationError) checkUint(param, paramName string) uint64 {
 	return num
 }
 
+func (v *validationError) checkAdConfigurationKeys(keysString string) []string {
```

Comment:
Will skip this as discussed, since generalizing the function parameter of `parseGroupIdList` to support arbitrary types and counts of arguments (such as the additional `*regexp.Regexp`) is not pleasantly applicable in golang.

Changed to a "light" version of this though, introducing a general `splitParameters` method